source/components/status-label/status-label.less

```diff
@@ -0,0 +1,56 @@
+hx-status {
+  background-color: transparent;
+  border: 1px solid;
+  border-color: @gray-750;
+  border-radius: 1em; // 10px
+  color: @gray-750;
+  display: inline-block;
```

Comment:
Since you're displaying as `inline-block`, you need to make sure that text doesn't wrap or you can get a status pill that breaks across two lines. Please add...
```less
white-space: nowrap;
```